content/cloud/noops-sql-db.md

```diff
@@ -0,0 +1,125 @@
+title = "NoOps SQLite Database"
+template = "cloud_main"
+date = "2023-07-26T16:00:00Z"
+enable_shortcodes = false
+[extra]
+url = "https://github.com/fermyon/developer/blob/main/content/cloud/noops-sql-db.md"
+
+---
+
+- [Accessing Private Beta](#accessing-private-beta)
+- [Service Limitations and Quotas For NoOps SQL Database in Fermyon Cloud](#service-limitations-and-quotas-for-noops-sql-database-in-fermyon-cloud)
+- [How To:](#how-to)
+	- [Prepare a NoOps SQL Database](#prepare-a-noops-sql-database)
+	- [Manage NoOps SQL Database Lifecycle](#manage-noops-sql-database-lifecycle)
+- [FAQ](#faq)
+- [Next Steps](#next-steps)
+
+With NoOps SQL Database using SQLite support in Fermyon Cloud, you can persist relational data generated by your Spin application in between application invocations without having to worry about database management. Fermyon Cloud will provision and manage the database on your behalf, no operations (NoOps) required. Fermyon Cloud will do the heavy lifting for you. To learn more about the Spin SQLite SDK, please visit the [API guide](../spin/sqlite-api-guide.md).
+
+<iframe width="854" height="480" src="https://www.youtube.com/embed/buwr66oRPbU" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
+
+## Accessing Private Beta
+
+NoOps SQL Database in Fermyon Cloud is currently in private beta. To request access to the private beta, please fill out this short [sign-up form](https://fibsu0jcu2g.typeform.com/to/Brv12FI0#hubspot_utk=xxxxx&hubspot_page_name=xxxxx&hubspot_page_url=xxxxx).
+ 
+> Please note that the private beta is limited in space and all requests cannot be guaranteed. 
+
+## Quotas And Service Limitations For NoOps SQL Database in Fermyon Cloud
+
+*Quotas* 
+* You can have a maximum of 1 NoOps SQL Database in your Cloud account
+* Your NoOps SQL Database can hold up 1 GB of data
+
+*Service Limitations*
+* The `--sqlite` flag not supported on `spin cloud deploy`
+* Fermyon Cloud only supports "default" NoOps SQL Databases at this time
```

Comment:
I'm not sure if this will make sense to everyone. Perhaps:

```suggestion
* Fermyon Cloud only supports NoOps SQLite Databases named "default" at this time
```